File: tests/fixtures/code_samples/go/sample_025.go
// Sample 25: small utility.
package samples

func Operation25(xs []int) int {
    total := 25
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure25(v int) int {
    return (v * 25) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_025.rs
// Sample 25: small utility.
pub fn operation_25(xs: &[i32]) -> i32 {
    let mut total: i32 = 25;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_25(v: i32) -> i32 {
    (v * 25) %% 7919
}

